src/components/sign-in-or-create/index.njk

```diff
@@ -68,15 +68,16 @@
     text: 'pages.signInOrCreate.createButtonText' | translate,
     value: "create",
     name: "optionSelected",
+    classes: "govuk-!-margin-right-3",
     attributes: {
       "id": "create-account-link"
     }
   }) }}
 
-    <p class="govuk-body">
-      {{ 'pages.signInOrCreate.paragraph2' | translate }}
-      <a href="/sign-in-or-create?redirectPost=true" class="govuk-link" id="sign-in-link">{{ 'pages.signInOrCreate.signInText' | translate }}</a>.
-    </p>
+    {{ govukButton({
+      text: 'pages.signInOrCreate.signInText' | translate,
+      classes: "govuk-button--secondary"
```

Comment:
The previous link for sign-in had an id. If I'm reading this correctly, the new button doesn't appear to have one. Could we have an id for it pls, unless there's a specific reason for not having one? (sign-in-button?)